scripts/docker/deploy/entrypoint.sh

```diff
@@ -8,6 +8,8 @@ fi
 AUTHENTICATION="true"
 
 # if command starts with an option, prepend arangod
+echo "PARAMETERS: "
+echo "$@"
```

Comment:
This looks like debug output. Do we want to keep it around permanently?
Same question for the other occurrence further down in this file.